src/components/Dashboard/Dashboard.js

```diff
@@ -34,6 +34,15 @@ import {
     isPrintMode,
 } from './dashboardModes'
 
+const setHeaderbarVisibility = mode => {
+    const header = document.getElementsByTagName('header')[0]
+    if (isPrintMode(mode)) {
+        header.classList.add('hidden')
+    } else {
+        header.classList.remove('hidden')
+    }
+}
+
```

Comment:
This should really be added as a feature of the platform/shell, rather than using this CSS hack.  (for another future change)

Reply:
I can add an issue for it?

Reply:
Thanks!  Maybe reference it in an issue on `app-platform`?

Reply:
Yes, will do.